### PR TITLE
Next prev feature

### PR DESCRIPTION
**Pull request by LesediMoko**
This pull request include the following:
Change of base url to the updated one
A next and previous button at the end of each chapter
An animation to show that the 'Trending' carousel is scrollable
A new clear all button to remove selected genres on Discovery page

### CODE REVIEW

File: src/modules/discover/components/GenreFilter.tsx [hunk 1]
```diff
@@ -24,6 +25,15 @@ const GenreFilter = () => {
 
     }
 
+    const clearGenreHandler = (values: { selectedOptions: string[] }) => {
+        console.log(useMangaStore.getState().selectedGenres)
```
Comment (yanevdw) [inline src/modules/discover/components/GenreFilter.tsx]: Please remove the console logs. 🔥

File: src/modules/discover/components/GenreFilter.tsx [hunk 2]
```diff
@@ -24,6 +25,15 @@ const GenreFilter = () => {
...
+        useMangaStore.setState({ selectedGenres: [] })
+        values.selectedOptions = []
+        console.log(useMangaStore.getState().selectedGenres)
```
Comment (yanevdw) [inline src/modules/discover/components/GenreFilter.tsx]: Here too. 👀

**Review by yanevdw — APPROVED** [review]
The chapter navigation looks great. Fantastic job!

**Review by Musa-Mabasa — APPROVED** [review]
Everything looks good. One note about the assertions, either than that, Well done 👍

**Pull request merged**